src/__tests__/components/ForecastTable.test.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { mount } from 'enzyme';
+
+import ForecastTable from '../../components/ForecastTable';
+
+const setup = () => {
+  const props = {
```

Comment:
`props` can be extracted as function or simple function, same with wrapper
`wrapper` can be `let` and you can use

```javascript
beforeEach(() => wrapper = mount())
```